server/lib/packets/login.go

```diff
@@ -3,44 +3,43 @@ package packets
 import (
 	"log"
 	
-	proto "../../proto"
-	"../models"
+	proto "github.com/umegaya/channer/server/proto"
+	"github.com/umegaya/channer/server/lib/models"
```

Comment:
__[gometalinter]__
* [gotype] could not import github.com/umegaya/channer/server/lib/models (can't find import: github.com/umegaya/channer/server/lib/models)